packages/twenty-server/src/database/typeorm/core/migrations/1733318043626-sso-missing-migration.ts

```diff
@@ -0,0 +1,177 @@
+    );
+    await queryRunner.query(
+      `ALTER TABLE "core"."workspaceSSOIdentityProvider" ALTER COLUMN "name" SET NOT NULL`,
+    );
+    await queryRunner.query(
+      `ALTER TABLE "core"."workspaceSSOIdentityProvider" DROP COLUMN "status"`,
+    );
+    await queryRunner.query(
+      `CREATE TYPE "core"."workspaceSSOIdentityProvider_status_enum" AS ENUM('Active', 'Inactive', 'Error')`,
+    );
+    await queryRunner.query(
+      `ALTER TABLE "core"."workspaceSSOIdentityProvider" ADD "status" "core"."workspaceSSOIdentityProvider_status_enum" NOT NULL DEFAULT 'Active'`,
+    );
+    await queryRunner.query(
+      `ALTER TYPE "core"."idp_type_enum" RENAME TO "idp_type_enum_old"`,
+    );
+    await queryRunner.query(
+      `CREATE TYPE "core"."workspaceSSOIdentityProvider_type_enum" AS ENUM('OIDC', 'SAML')`,
+    );
+    await queryRunner.query(
+      `ALTER TABLE "core"."workspaceSSOIdentityProvider" ALTER COLUMN "type" DROP DEFAULT`,
+    );
+    await queryRunner.query(
+      `ALTER TABLE "core"."workspaceSSOIdentityProvider" ALTER COLUMN "type" TYPE "core"."workspaceSSOIdentityProvider_type_enum" USING "type"::"text"::"core"."workspaceSSOIdentityProvider_type_enum"`,
+    );
+    await queryRunner.query(
+      `ALTER TABLE "core"."workspaceSSOIdentityProvider" ALTER COLUMN "type" SET DEFAULT 'OIDC'`,
+    );
+    await queryRunner.query(`DROP TYPE "core"."idp_type_enum_old"`);
+    await queryRunner.query(
+      `ALTER TYPE "core"."workspace_activationStatus_enum" RENAME TO "workspace_activationStatus_enum_old"`,
+    );
+    await queryRunner.query(
+      `CREATE TYPE "core"."workspace_activationstatus_enum" AS ENUM('ONGOING_CREATION', 'PENDING_CREATION', 'ACTIVE', 'INACTIVE')`,
+    );
+    await queryRunner.query(
+      `ALTER TABLE "core"."workspace" ALTER COLUMN "activationStatus" DROP DEFAULT`,
+    );
+    await queryRunner.query(
+      `ALTER TABLE "core"."workspace" ALTER COLUMN "activationStatus" TYPE "core"."workspace_activationstatus_enum" USING "activationStatus"::"text"::"core"."workspace_activationstatus_enum"`,
+    );
+    await queryRunner.query(
+      `ALTER TABLE "core"."workspace" ALTER COLUMN "activationStatus" SET DEFAULT 'INACTIVE'`,
+    );
+    await queryRunner.query(
+      `DROP TYPE "core"."workspace_activationStatus_enum_old"`,
+    );
+    await queryRunner.query(
+      `ALTER TABLE "core"."workspace" ALTER COLUMN "isGoogleAuthEnabled" SET NOT NULL`,
+    );
+    await queryRunner.query(
+      `ALTER TABLE "core"."workspace" ALTER COLUMN "isPasswordAuthEnabled" SET NOT NULL`,
+    );
+    await queryRunner.query(
+      `ALTER TABLE "core"."workspace" ALTER COLUMN "isMicrosoftAuthEnabled" SET NOT NULL`,
+    );
+    await queryRunner.query(
+      `ALTER TABLE "core"."workspaceSSOIdentityProvider" ADD CONSTRAINT "FK_bc8d8855198de1fbc32fba8df93" FOREIGN KEY ("workspaceId") REFERENCES "core"."workspace"("id") ON DELETE CASCADE ON UPDATE NO ACTION`,
+    );
```

Comment:
These two constraints are usefull and shouldn't be dropped:

```ts
await queryRunner.query(
      `ALTER TABLE "core"."workspaceSSOIdentityProvider" ADD CONSTRAINT "CHK_SAML" CHECK ((((type = 'SAML'::core.idp_type_enum) AND ("ssoURL" IS NOT NULL) AND (certificate IS NOT NULL)) OR (type = 'OIDC'::core.idp_type_enum)))`,
    );
    await queryRunner.query(
      `ALTER TABLE "core"."workspaceSSOIdentityProvider" ADD CONSTRAINT "CHK_OIDC" CHECK ((((type = 'OIDC'::core.idp_type_enum) AND ("clientID" IS NOT NULL) AND ("clientSecret" IS NOT NULL)) OR (type = 'SAML'::core.idp_type_enum)))`,
    );
    ```

Reply:
After discussion offline, let's remove those for now since typeorm does not handle Check annotation with expressions and take time to find a better implementation later